lnwallet/interface_test.go

```diff
@@ -114,6 +114,22 @@ func assertReservationDeleted(res *lnwallet.ChannelReservation, t *testing.T) {
 	}
 }
 
+// assertRecoveryInfoMatch asserts that the supplied recovery info is correct.
+func assertRecoveryInfoMatch(t *testing.T,
+	expectedRecoveryMode, isRecoveryMode bool,
+	expectedProgress, progress float32) {
+	t.Helper()
+
+	if isRecoveryMode != expectedRecoveryMode {
```

Comment:
nit: we've started using the `testify/require` to do handle our test assertions. not blocking, as this code already works, but you could consider using `require.Equal`. up to you if you would like to switch the whole test over to `require` and use the other variants `require.Nil`, `require.Error`, etc :)

Reply:
Switched. I prefer `require` as it's much cleaner ;)